src/test/java/seedu/address/ui/MemberCardTest.java

```diff
@@ -35,7 +35,7 @@ public void equals() {
         assertTrue(memberCard.equals(memberCard));
 
         // null -> returns false
-        assertFalse(memberCard.equals(null));
+        assertFalse(memberCard == null);
```

Comment:
I think the `.equals()` here is correct- it is meant to check if the `.equals() ` function works correctly on `null` objects.

Reply:
fixed